### DISCUSSION TITLE
False errors in multi-files model

### DISCUSSION THREAD

**Discussion by chapuisk** [gama-platform/gama#121] [Q&A]
Hi there,

Using the 1.9.3 release, I notice that the amount of "false errors" had increased a lot. What I called "false error" is error displayed in the model view, while nothing wrong in the code. See in the screenshot that test2.gaml is showing error on file entreprise.gaml, while this file does not have any error:

![Screenshot 2024-03-12 at 10 56 36](https://github.com/gama-platform/gama/assets/13599506/0c026044-ecd4-4772-96cb-7de160cd9b24)

It is pretty annoying, even if those errors do not prevent the model to run. To get ride of those errors, I do have several tricks: close & reopen gaml files, comment & uncomment lines with errors or close re-open project. I suppose that it is linked to xtext & eclipse, hence hard to cope with, but from my experience it has increased / turned systematic from 1.9.2 to 1.9.3. Have you noticed the same on your side?    

Kevin

**Comment by lesquoyb** [comment]
I've notice this sometimes too, but not systematically, do you have a model to provide so we can test that ?

**Reply by chapuisk** [reply]
I did not play with comokit since a long time, but I think that there is enough complexity over there to test any multi-files issues :)

**Comment by lesquoyb** [comment]
I turned it into an issue so closing the discussion, we'll talk about the details there
